app/actions/sections.ts

```diff
@@ -20,3 +20,5 @@ export const TeamSection = ({ t }: ActionContext) => t("Workspace");
 
 export const RecentSearchesSection = ({ t }: ActionContext) =>
   t("Recent searches");
+
+export const TrashSection = ({ t }: ActionContext) => t("Trash");
```

Comment:
I don't think this is needed either, it's fine in the DocumentsSection?

Reply:
Since we are acting on Trash, I thought creating a new section with its associated action(s) provided a clean separation (and also a clean UI in the cmd-bar).
I'm not sure `Empty Trash` is something that needs to be associated to a single document (and hence in the Documents section of cmd-bar).

Reply:
Okay, sure – lets keep the section but I'd still move the action into `definitions/documents` as definitely acting on documents fundamentally.

Reply:
Sure, will do.